splunk-otel-android/src/main/java/com/splunk/rum/ActivityCallbacks.java

```diff
@@ -169,28 +169,20 @@ public void onActivityDestroyed(@NonNull Activity activity) {
 
     @Override
     public void onActivityPostDestroyed(@NonNull Activity activity) {
-        getActivityTracer(activity).addEvent("activityPostDestroyed").endActiveSpan();
+        getTracer(activity).addEvent("activityPostDestroyed").endActiveSpan();
     }
 
     private void addEvent(@NonNull Activity activity, String eventName) {
-        getActivityTracer(activity).addEvent(eventName);
+        getTracer(activity).addEvent(eventName);
     }
 
-    private TrackableTracer getOrCreateTracer(Activity activity) {
-        NamedTrackableTracer activityTracer = tracersByActivityClassName.get(activity.getClass().getName());
+    private ActivityTracer getTracer(Activity activity) {
```

Comment:
I actually intended to rename them all to `getTracer` in all places and just forgot. This is because I got rid of the method that didn't create. Do you think I should keep the `getOrCreate...` name everywhere?

Reply:
I've renamed them all to `getTracer` for now, but happy to rename back if you think that's better. (I consider the "getOrCreate" an implementation detail now that we always create if necessary)